profiles/models.py

```diff
@@ -0,0 +1,28 @@
+"""
+Models for user profile
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class Profile(models.Model):
+    """
+    A user profile
+    """
+    user = models.ForeignKey(User)
+    edx_username = models.TextField()
```

Comment:
why is this here? can it be even possible to have a different username from edx?